File: velosystem-velocity/src/main/java/eu/moonwriters/velosystemvelocity/util/CommandMetaUtil.java
package dev.eztxm.velosystem.velocity.util;

import com.velocitypowered.api.command.CommandManager;
import com.velocitypowered.api.command.CommandMeta;

public class CommandMetaUtil {
    private final Object plugin;
    private final CommandManager commandManager;

    public CommandMetaUtil(Object plugin, CommandManager commandManager) {
        this.plugin = plugin;
        this.commandManager = commandManager;
    }

    public CommandMeta create(String commandName, String... aliases) {
        return commandManager.metaBuilder(commandName)
                .aliases(aliases)
                .plugin(plugin)
                .build();
    }
}

File: velosystem-velocity/src/main/kotlin/eu/moonwriters/velosystemvelocity/util/CommandMetaUtil.kt
package eu.moonwriters.velosystemvelocity.util

import com.velocitypowered.api.command.CommandManager
import com.velocitypowered.api.command.CommandMeta

class CommandMetaUtil(private val plugin: Any, private val commandManager: CommandManager) {

    fun create(commandName: String?, vararg aliases: String?): CommandMeta {
        return commandManager.metaBuilder(commandName)
            .aliases(*aliases)
            .plugin(plugin)
            .build()
    }
}
